Translators/WZDx/main.py

```diff
@@ -82,3 +85,45 @@ def translateWzdxTIM(request):
 
     tims = translate(request.get_json())
     return (json.dumps(tims), 200, headers)
+
+def entry(request):
+    logging.info('TIM Translator Timer Called...')
+    if request.method == 'OPTIONS':
+        headers = {
+            'Access-Control-Allow-Origin': '*',
+            'Access-Control-Allow-Methods': 'POST',
+            'Access-Control-Allow-Headers': 'Content-Type',
+            'Access-Control-Max-Age': '3600'
+        }
+
+        return ('', 204, headers)
+
+    headers = {
+        'Access-Control-Allow-Origin': '*'
+    }
+
+    # Scrape the CDOT endpoint to get current list of WZDX features
+    geoJSON =  json.loads(requests.get(f'https://{os.getenv("WZDX_ENDPOINT")}/api/v1/wzdx?apiKey={os.getenv("WZDX_API_KEY")}').content.decode('utf-8'))
+    
+    tim_list = translate(geoJSON)
+
+    logging.info('Pushing TIMs to ODE...')
+
+    errNo = 0
+    for tim in tim_list:
+        for msg in tim["tim"]["dataframes"]:
+            # change 'workzone' to 'workZone' before pushing to ODE
+            if msg["content"] == "workzone":
+                msg["content"] = "workZone"
+            # update start date to include milliseconds if missing
+            if msg["startDateTime"][-5] != ".":
+                msg["startDateTime"] = msg["startDateTime"][:-1] + ".000Z"
+        return_value = requests.post(f'{os.getenv("TIM_TIMER_URL")}/tim', json=tim)

```

Comment:
What is this TIM_TIMER_URL supposed to be? Is this just the ODE endpoint?

Reply:
Yes, this is just the ODE endpoint

Reply:
Should we rename it to be more precise and reduce confusion?

Reply:
Let's also add any needed variables to the README so we know what's required when coming into the program.

Reply:
Changed variable name from TIM_TIMER_URL to ODE_ENDPOINT, updated README to include necessary WZDx environment variables to scrape the WZDx endpoint